### ISSUE TITLE
redirect_uri_mismatch

### ISSUE THREAD

**Issue by pavelsr** [rjwhite/Google-Calendar-Perl-access#2]
Unfortunately your Bourne shell script (oauth2.sh) wouldn't work with new API v3

According note in [console](https://console.developers.google.com/apis/credentials) [RedirectURI that your are using](https://github.com/rjwhite/Google-Calendar-Perl-access/blob/master/oauth2.sh#L24) isn't allowed

**Comment by rjwhite** [comment]
Hi Pavel,
I am unable to duplicate whatever problem you ran into.
I created a new calendar from scratch.  I created new oauth2 credentials from scratch (Client ID and Client secret), and ran my oauth2.sh program, twice,  as per the comments at the top and was successfully given a refresh code.
I then successfully used my sample create_cal_entry_full.plx program to book an appointment in my new calendar.
This was done on a Linux Mint 17 Qiana system.
